client/src/assets/partials/lesson.html

```diff
@@ -66,6 +68,10 @@ <h3>Level {{selectedLevel}}</h3>
 
 <div class="divider"></div>
 
+<div class="lesson-end" ng-show="lesson.state === 'Running'">
```

Comment:
Don't compare vs strings when we have enum values - strings are very brittle.
Better yet, `lesson` can provide a helper method like `lesson.isRunning()`

Reply:
Helper method makes sense.